app/views/articles/_single_story.html.erb

```diff
@@ -3,8 +3,9 @@
     <div id="featured-story-marker" data-featured-article="articles-<%= story.id %>"></div>
   <% end %>
   <% if featured == true || (feed_style_preference == "rich" && story.main_image.present?) %>
-    <a href="<%= story.path %>" class="crayons-story__cover" aria-label="article">
-      <div style="background-color: <%= story.main_image_background_hex_color %>; background-image: url(<%= cloud_cover_url(story.main_image) %>);" class="crayons-story__cover__image"></div>
+    <a href="<%= story.path %>" title="<%= story.title %>" class="crayons-story__cover" aria-label="article"
```

Comment:
Logged out view of the featured post. Hidden text so that screen readers are all good.